.github/workflows/docker-image.yml

```diff
@@ -21,4 +32,4 @@ jobs:
         uses: docker/build-push-action@v2
         with:
           push: true
-          tags: jokkedk/webgrind:latest
```

Comment:
Thank you for the PR. I think we should keep the :latest tag so it is possible to always target this

Reply:
I believe that the docker-meta-action tag generation automatically includes a latest tag for whatever version it's building. I definitely could be wrong about that though and I can't find a good clarification about it.

Perhaps if it does not include the latest tag, per https://github.com/docker/build-push-action#customizing we can just put a comma separated list there, and from some other examples seems like this setup should work.